packages/web-components/src/components/va-search/va-search.tsx

```diff
@@ -0,0 +1,43 @@
+import { Component, Host, h, Prop, Event, EventEmitter } from '@stencil/core';
+
+@Component({
+  tag: 'va-search',
+  styleUrl: 'va-search.css',
+  shadow: true,
+})
+export class VaSearch {
+  @Prop() action: string;
```

Comment:
Missing the jsdoc comments could prop just copy the definitions: https://www.w3schools.com/tags/tag_form.asp